src/ceph-disk/run-tox.sh

```diff
@@ -17,6 +17,12 @@
 
 # run from the ceph-disk directory or from its parent
 : ${CEPH_DISK_VIRTUALENV:=/tmp/ceph-disk-virtualenv}
+if [ -z $CEPH_BIN ]; then
```

Comment:
@tchaikov 
It allows it to run, which helps a lot.
But ceph-disk will require quite a lot of low-level device code to actually match what is available as devices in FreeBSD.
So perhaps I should set this to DNM? I just did not want to lose this code, and get some others want to help FreeBSD porting to get going.